<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="SHREYAS" failures="2" tests="6" name="Test" time="50.142" errors="0" timestamp="2024-11-26T16:12:01 IST">
  <testcase classname="testCases.TC_01" name="verify_account_registation" time="5.035"/>
  <testcase classname="testCases.TC_02" name="verify_login" time="7.79"/>
  <testcase classname="testCases.TC_02_DDT" name="verify_loginDDT" time="5.419"/>
  <testcase classname="testCases.TC_02_DDT" name="verify_loginDDT" time="7.689"/>
  <testcase classname="testCases.TC_02_DDT" name="verify_loginDDT" time="2.235">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at testCases.TC_02_DDT.verify_loginDDT(TC_02_DDT.java:56)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 35 stack frames]]>
    </failure>
  </testcase> <!-- verify_loginDDT -->
  <testcase classname="testCases.TC_02_DDT" name="verify_loginDDT" time="4.647">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at testCases.TC_02_DDT.verify_loginDDT(TC_02_DDT.java:47)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 35 stack frames]]>
    </failure>
  </testcase> <!-- verify_loginDDT -->
</testsuite> <!-- Test -->
